examples/flow/flow_pois.py

```diff
@@ -58,6 +61,33 @@ def get_set(file_name):
 wrapper = LammpsWrapper()
 wrapper.CM[CUBA.NUMBEROF_TIME_STEPS] = 10000
 wrapper.CM[CUBA.TIME_STEP] = 0.003
+wrapper.CM[CUBA.PAIR_POTENTIALS] = ("lj:\n"
```

Comment:
> I don’t understand the question. In general, the user can configure the PAIR_POTENTIALS for the SP according to his/her problem.

Sorry for the confusion the question wonders if one could provide a yml filename for the PAIR_POTENTIALS value which would limit the user to selecting one of the available files instead of allowing the user to create the pairs dynamically at runtime (as discussed in the last question).

Thanks for the quick response.